src/helpers/ukef-id.type.ts

```diff
@@ -0,0 +1,9 @@
+// Start of id differs depending on environment
+type DealFacilityPartyUkefIdStart = '0020' | '0030' | '0040';
```

Comment:
Can these please loaded from centralised constants?

Reply:
Can the constants be stored in environment specific naming convention.
`DEV: 0020`...

Reply:
> Can these please loaded from centralised constants?

I'm not sure if they are repeated anywhere else? constants should be useful if they are used in multiple places

Reply:
They are just common start of ID, prefix. 
Not sure if they will be reused, maybe in generation/active test code. So will move to constants.

Reply:
Looks like Typescript String Literal Types doesn't support imports, or maybe constants, so leaving as strings.